src/Microsoft.DotNet.XHarness.iOS/AppRunner.cs

```diff
@@ -230,23 +230,22 @@ public class AppRunner : AppRunnerBase
 
                 _mainLog.WriteLine("Starting test run");
 
-                await _processManager.ExecuteCommandAsync(
+                var result = await _processManager.ExecuteCommandAsync(
                     mlaunchArguments,
                     _mainLog,
                     timeout,
                     cancellationToken: cancellationToken);
+
+                if (!result.Succeeded)
+                {
+                    throw new Exception("Failed to run the application!");
+                }
             }
             finally
             {
                 deviceLogCapturer.StopCapture();
                 deviceSystemLog.Dispose();
             }
-
-            // Upload the system log
-            if (File.Exists(deviceSystemLog.FullPath))
```

Comment:
Why was this removed? Isn't it still needed?

Reply:
Yeah, it was useless